src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const selection = people.filter(
```

Comment:
You have to be more specific here

Reply:
👌fixed
All `selection`s renamed to `men`, `women`, `hasMotherData` depending on the appropriate function